packages/components/src/Select/intl.ts

```diff
@@ -1,8 +0,0 @@
-export const messages = {
```

Comment:
why is this deleted?

Reply:
we did not use this for a long time (was used when we used hooks)